includes/utils.php

```diff
@@ -68,7 +100,7 @@ function delete_autoshare_for_twitter_meta( $id, $key ) {
  * @return bool
  */
 function maybe_autoshare( $post_id ) {
-	return ( 1 === intval( get_autoshare_for_twitter_meta( $post_id, ENABLE_AUTOSHARE_FOR_TWITTER_KEY ) ) ) ? true : false;
+	return ( 1 === intval( autoshare_enabled( $post_id ) ) ) ? true : false;
```

Comment:
Should we remove this function? `maybe_autoshare()` is only used in `post-transition.php:88`. And it's just a wrapper for `autoshare_enabled`.

Reply:
Yes, good idea. I have updated in https://github.com/10up/autoshare-for-twitter/pull/71/commits/bf7f4444d9e8366977c64179a9052d72cfaa86fa